src/libraries/Common/src/Interop/Windows/Kernel32/Interop.GetCommProperties.cs

```diff
@@ -8,8 +8,8 @@ internal static partial class Interop
 {
     internal static partial class Kernel32
     {
-        [DllImport(Libraries.Kernel32, SetLastError = true, CharSet = CharSet.Auto)]
-        internal static extern bool GetCommProperties(
+        [GeneratedDllImport(Libraries.Kernel32, CharSet = CharSet.Unicode, SetLastError = true)]
```

Comment:
Why is Unicode needed here... is that because COMMPROP contains a char?

Reply:
Should not be needed - removed.